Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN  chown -R app: /home/app
 ENV HOME /home/app
 WORKDIR /home/app
 ENV APP_HOME /home/app
-USER app
+USER 1000
```

Comment:
Since we hard-set `--uid 1000` for the `app` user, do we need to change `USER app`?

Reply:
It shouldn't make a difference, right? But it does here 😱 I don't know why, but live-1 wouldn't let it run until I made this change

Reply:
Ah... https://user-guide.cloud-platform.service.justice.gov.uk/tasks.html#how-to-adapt-to-the-pod-security-policies

> **You must specify the user by its numeric UID**, as above, not by its username. If you use the username (`USER myNewUser`) then the pod security policy will not be able to tell that that is a non-root user, and your container will not be scheduled.

That is annoying.

Reply:
💭 Since this is one the things which I would immediately attempt to change if I stumble upon it in Dockerfiles, could we add a comment above it saying we need this in numeric format, please?